src/datasets/sdks/nextjs-flags-sdk.ts

```diff
@@ -0,0 +1,16 @@
+import { SDK } from '.';
+
+export const NestjsFlagsSDK: SDK = {
+  name: 'Next.js Flags',
+  description: 'Vercel Next.js Flags SDK with OpenFeature Adapter',
+  filename: 'javascript/nextjs-flags-sdk',
+  category: 'Client',
```

Comment:
Labeling for both Server and Client would be great 🙏

Reply:
Supporting both will require refactoring all the data models here, so I think we will just need to choose one for now.

Reply:
What a timing, I just started looking into it myself and noticed it would be a bigger thing. I agree. Keeping client for now sounds good 🙏